.github/workflows/build.yml

```diff
@@ -2,10 +2,14 @@ name: Build
 on:
   pull_request:
   workflow_dispatch:
+  push:
+    branches:
+      - dev
```

Comment:
won't this break builds for prs?

Reply:
It won't - per the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-activity-types-and-filters-with-multiple-events) the whole `on` object means "trigger on one of (PR, manual dispatch, push to `dev`)"

In fact - this PR's CI [already runs off of the mentioned file](https://github.com/PGMDev/PGM/actions/runs/10024239030/workflow?pr=1367)!